docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -16,47 +16,196 @@ redirect_from:
+
+Though etcd keeps unique member ID internally, it is recommended to use a unique name for each member to avoid human errors. For example, consider a three-member etcd cluster. Let the URLs be, member1=http://10.0.0.1, member2=http://10.0.0.2, and member3=http://10.0.0.3. When member1 fails, replace it with member4=http://10.0.0.4.
+
+1. Get the member ID of the failed member1:
+
+    `etcdctl --endpoints=http://10.0.0.2,http://10.0.0.3 member list`
+
+      The following message is displayed:
+
+        8211f1d0f64f3269, started, member1, http://10.0.0.1:12380, http://10.0.0.1:2379
+        91bc3c398fb3c146, started, member2, http://10.0.0.1:2380, http://10.0.0.2:2379
+        fd422379fda50e48, started, member3, http://10.0.0.1:2380, http://10.0.0.3:2379
 
-## etcd for Kubernetes: high-level goals
+2. Remove the failed member:
 
-Access control: Give only kube-apiserver read/write access to etcd. You do not
-want the API server's etcd exposed to every node in your cluster, or worse, to the
-internet at large. Access to etcd is equivalent to root access in your
-cluster.
+    `etcdctl member remove 8211f1d0f64f3269`
 
-Data reliability: For reasonable safety, either etcd needs to be run as a
-[cluster](/docs/admin/high-availability/#clustering-etcd), or etcd's data directory
-should be located on durable storage, for example, a Google Compute Engine persistent disk.
-In either case, if high availability is
-required--as it might be in a production cluster--the data directory ought to be
-[backed up periodically](https://coreos.com/etcd/docs/latest/op-guide/recovery.html)
-to reduce downtime in case of corruption.
+      The following message is displayed:
+
+       Removed member 8211f1d0f64f3269 from cluster
+
+3. Add the new member:
+
+    `./etcdctl member add member4 --peer-urls=http://10.0.0.4:2380`
+
+     The following message is displayed:
+
+       Member 2be1eb8f84b7f63e added to cluster ef37ad9dc622a7c4
+
+4. Start the newly added member on a machine with the IP `10.0.0.4`:
+
+    ```bash
+    export ETCD_NAME="member4"
+    export ETCD_INITIAL_CLUSTER="member2=http://10.0.0.2:2380,member3=http://10.0.0.3:2380,member4=http://10.0.0.4:2380"
+    export ETCD_INITIAL_CLUSTER_STATE=existing
+    etcd [flags]
+    ```
+5. Do either of the following:
+
+   1. Update its `--etcd-servers` flag to make Kubernetes aware of the configuration changes, then restart the Kubernetes API server.
+   2. Update the load balancer configuration if a load balancer is used in the deployment.
+
+For more information on cluster reconfiguration, see [etcd Reconfiguration Documentation](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/runtime-configuration.md#remove-a-member).
+
+## Taking a backup of an etcd cluster
```

Comment:
suggested edit: "Backing up an etcd cluster" or "Taking a snapshot of an etcd cluster"